test/built-ins/Temporal/PlainDateTime/prototype/withCalendar/calendar-invalid-iso-string.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.withcalendar
+description: Invalid ISO 8601 string is not accepted as calendar
+features: [Temporal]
+---*/
+
+const instance = new Temporal.PlainDateTime(1976, 11, 18, 15, 23, 30, 123, 456, 789, "iso8601");
+
+const invalidStrings = [
+  ["", "empty string"],
+  ["1997-12-04[u-ca=iso8601]", "ISO string with calendar annotation"],
```

Comment:
I think this line should not be here. According to my reading of [the proposal](https://tc39.es/proposal-temporal/#sec-temporal.plaindate.prototype.withcalendar), it's valid to pass an ISO string to this method.